var/spack/repos/builtin/packages/root/package.py

```diff
@@ -505,7 +513,6 @@ def cmake_args(self):
             define_from_variant("gl2ps", "opengl"),
             define("glite", False),
             define("globus", False),
-            define_from_variant("gsl_shared", "gsl"),
```

Comment:
Just to confirm: removing this option for *all* version installs was deliberate, correct?

Reply:
It never did anything.

Reply:
Not in this millenium anyway.